Userland/Applications/Piano/Music.h

```diff
@@ -155,19 +155,19 @@ constexpr int beats_per_bar = 4;
 constexpr int notes_per_beat = 4;
 constexpr int roll_length = (sample_rate / (beats_per_minute / 60)) * beats_per_bar;
 
-constexpr char const* note_names[] = {
-    "C",
-    "C#",
-    "D",
-    "D#",
-    "E",
-    "F",
-    "F#",
-    "G",
-    "G#",
-    "A",
-    "A#",
-    "B",
+constexpr StringView note_names[] = {
```

Comment:
It would be nice to split the "non-trivial" SV changes into their own commits, like this one + the change in RollWidget, to make the review easier ^^

Reply:
oh no, someone's touching Piano :skeleyak: